src/generic/grid.cpp

```diff
@@ -5876,6 +5877,18 @@ void wxGrid::OnKeyDown( wxKeyEvent& event )
                 }
                 break;
 
+            case 'C':
+                if (event.GetModifiers() == wxMOD_CONTROL)
+                {
+                    wxGridCellCoordsArray selections = GetSelectedCells();
+                    if (selections.size() == 1 && wxTheClipboard->Open())
+                    {
+                        wxTheClipboard->SetData(new wxTextDataObject(GetCellValue(selections[0])));
+                        wxTheClipboard->Close();
+                    }
+                }
```

Comment:
Probably doesn't change much in practice, but there should be `else { event.Skip(); }` here.